nss-day-cohort-45/nutyshell-running-with-scissors-1
# Rcd event edit

# Description
Added an edit button for each event that a user has saved (does not appear on friend events)

## Type of change

# Testing Instructions
-Run JSON
-Run Serve
-Double check add and delete functionality for the user's events
-See if friend events show up and that they do not allow the user to edit or delete them
-See if you can click on the edit button and change any user event

# Checklist:
